packages/manager/modules/web-paas/src/terminate/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerWebPaasTerminateLazyloading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('web-paas.cancel.**', {
+      url: '/:projectId/cancel',
```

Comment:
```suggestion
      url: '/cancel',
```
`:projectId` shouldn't be duplicated, the state should be a child of the one declaring `/:projectId` url

Reply:
This terminate is called directly from the list page, hence have added `projectId` in the URL here.

Reply:
If I'm on a listing page let's say `product.index`, nothing prevents me from going to `product.dashboard.terminate`